### ISSUE TITLE
handle InvalidTokenError and fix setProperty

### ISSUE THREAD

**Issue by williwacker** [openviess/PyViCare#315]
see fixes in #314 

With the latest release I am getting "InvalidTokenError" after some time  and I can no longer set any values

**Comment by vielleicht** [comment]
I just implemented some workaround, but I also would prefer the library handling that exception.

```
    vicare = PyViCare()
    retry = 3
    while retry > 0:
        try:
            vicare.initWithCredentials("xxx","xxx", client_id="xxx", token_file="token.save")
            break
        except authlib.integrations.base_client.errors.InvalidTokenError as ex:
            if os.path.exists("token.save"):
                print("invalid token '%s' -> removing old token" % ex)
                os.remove("token.save")
            else:
                print("oh, token already removed, this will fail...")
        retry -= 1
```

**Comment by crazyfx1** [comment]
Was fixed by @williwacker in #314
